### PR TITLE
Update readme to cover basic workflows

### PR DESCRIPTION
**Pull request by ChrisBr**

### CODE REVIEW

**Comment by ChrisBr** [comment]
@hennevogel @cornelius please review

File: README.md [hunk 1]
```diff
@@ -1,3 +1,79 @@
...
+  
+  **Get started:** Tasks that mentors may want to suggest students so that they can start contributing to the code base (e.g. junior jobs, low hanging fruits, discussion on the mailing list)
+
+3. Label the issue with your projects name (e.g. Machinery) and assign it to yourself. It will now magically appear on our [project site]()!
+
+4. Wait till a Mentee gets in touch with you! 
+
+## Mentee
+
+You want to start working on a mentoring project? Awesome! Just pick one of the proposed project from our issue list and follow the get started instruction. Please also write a short comment in the issue that you would like to work on this project.
```
Comment (cornelius) [inline README.md]: Should it be "working on a mentored project"?

**Comment by hennevogel** [comment]
Why do you add this to the readme and not to the web page? :-)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,79 @@
...
+
+### Google Summer of Code
```
Comment (cornelius) [inline README.md]: Maybe the GSoC instructions should better be on the web page.

We could add a generic description in the README saying something like "Google and others are running mentoring programs such as the Google Summer of Code. You can find instructions how to work on openSUSE projects there on the [mentoring web site](appropriate link). But you can always get involved if you find a project interesting. Just get in contact with the mentor by commenting on the GitHub issue of the project."

File: README.md (hunk 1 above)
Comment (ChrisBr) [inline README.md]: Ok!

**Comment by ChrisBr** [comment]
@cornelius as discussed with @hennevogel in person: We will move the templates to their own files and link to them (mentor_template, gsoc_template)! In the readme we will only cover how to contribute to the project.

**Comment by ChrisBr** [comment]
I'll close this for now

**Pull request closed without merging**